sky/utils/accelerator_registry.py

```diff
@@ -1,9 +1,21 @@
-"""Resources related utilities."""
+"""Accelerator registry."""
 from sky.clouds import service_catalog
 from sky.utils import ux_utils
 
-# List of all accelerators supported by SkyPilot.
+# Canonicalized names of all accelerators (except TPUs) supported by SkyPilot.
 # NOTE: Must include accelerators supported for local clusters.
+
```

Comment:
styleguide nit: L7 should have a `#`, and L18 should not be blank --  to form a block of comments from L5 to L17 for L19.

Reply:
Thanks for letting me know! Fixed.